### DISCUSSION TITLE
Create Swap partition upon SL installation?

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#463] [Q&A] [answered]
Hey folks. I'm currently installing Spiral Linux.

I'm a dumb dumb and forgot whether or not I need to create a Swap partition in the Calamares Installer or to just leave it blank and choose "No Swap".

Normally, I create a Swap partition on all my Linux systems and set the snappiness to like 20 or 10.

But y'all are using zRAM. How do I get the zRAM benefit? Create Swap partition or leave it as "No Swap"

**Comment by AlphaElwedritsch** [comment, marked as answer]
Leave "no swap" will create a zRAM swap file.
Can be checked by `sudo swap on`

**Reply by geckolinux** [reply]
> Leave "no swap" will create a zRAM swap file.

Just a small correction, zRAM isn't a swap _file_, but a RAM compression technique. It will be enabled by default in SpiralLinux regardless of which swap choice is selected in the Calamares installer.
